tasties_app/tests/test_recipes_view.py

```diff
@@ -34,6 +36,20 @@ def test_view_recipe_not_found(self, client, signed_up_credentials, login_to_sit
         assert response.status_code == 302
         assert response.url == '/'
 
+    def test_add_comment_function(self, client, comment, recipe_test, create_user):
+        client.force_login(create_user)
+        response = client.get(f'/view_recipe/{recipe_test.id}/?comment-adding={comment.comment_text}')
+        assert response.status_code == 200
+        comment_test = Comment.objects.get(author_id=comment.author_id, comment_text=comment.comment_text)
```

Comment:
You can `Comment.objects.get(pk=comment.id)`, there's no reason for you to `.get` an object here by looking for something that can have multiple meanings (Think, what would happen, if some adversary plastered the same comment all over a number of different recipes? What if that adversary was one of us doing this in a data migration somewhere down the road?).